src/QMCDrivers/QMCCostFunctionBase.h

```diff
@@ -77,6 +77,10 @@ class QMCCostFunctionBase : public CostFunctionBase<QMCTraits::RealType>, public
   ///process xml node
   bool put(xmlNodePtr cur);
   void resetCostFunction(std::vector<xmlNodePtr>& cset);
+  ///Save opt parameters to HDF5
+  bool reportH5;  
+  bool CI_Opt;  
+  char newh5[128];
```

Comment:
Need comments on both CI_Opt and newh5. newh5 better to be replaced with std::string.